ui/src/publication/table/publication-table.tsx

```diff
@@ -47,12 +49,36 @@ const PublicationTable: React.FC<PublicationTableProps> = ({
         </Th>
     );
 
+    const [itemDetailsVisible, setItemDetailsVisible] = useState<PublicationTableItem['id'][]>([]);
+
+    const publicationItemDetailsVisibilityToggle = (id: PublicationTableItem['id']) => {
+        if (!itemDetailsVisible.includes(id)) {
+            setItemDetailsVisible([...itemDetailsVisible, id]);
+        } else {
+            setItemDetailsVisible(itemDetailsVisible.filter((existingId) => existingId !== id));
+        }
+    };
+
+    const anyPublicationItemDetailsVisible = itemDetailsVisible.length > 0;
+    const toggleVisibilityOfAllDetails = () => {
+        if (anyPublicationItemDetailsVisible) {
+            setItemDetailsVisible([]);
+        } else {
+            setItemDetailsVisible(items.map((item) => item.id));
+        }
+    };
+
     return (
         <div className={styles['publication-table']}>
             <Table wide isLoading={isLoading}>
                 <thead className={styles['publication-table__table-header']}>
                     <tr>
-                        <Th />
+                        <Th>
+                            <AccordionToggle
+                                open={anyPublicationItemDetailsVisible}
+                                onToggle={() => toggleVisibilityOfAllDetails()}
+                            />
```

Comment:
Toiminnallisuutena hyvä, rupesin vaan miettimään noita ikoneita 🤔 Monissa järjestelmissä tällaiset "expand/collapse all" -toiminnallisuudet on merkitty omilla ikoneillaan. Esim. IDEA käyttää tämän kuvan kaltaisia:

<img width="56" alt="image" src="https://github.com/user-attachments/assets/cf8c7fab-82c3-4133-af82-0aeed2f7c6ae">

Figmassa näyttäisi löytyvän vähän niinkuin tuon tyyppiset ikonit (käännetty 45 astetta.) Voisi ehkä harkita niiden käyttöä tähän

Reply:
Ehkäpä 🤔, tosin henk. koht tykkään siitä minimalismista että ovat samoja. Tuollainen saattaisi myös kiinnittää aika paljon huomiota vasemmassa yläkulmassa, johon käyttäjän silmät osuvan aina ensimmäisenä näin länsimaissa